lib/worker.coffee

```diff
@@ -44,3 +51,25 @@ module.exports = (text, attributes, options) ->
       stack: stack
 
   callback()
+
+registerBlocksPositions = (doc) ->
+  # Use Ruby API:
+  # because `doc.$find_by()` doesn't accept a filter function as parameter.
```

Comment:
It's more correct to say "because `doc.findBy()` doesn't yet accept a filter function as a parameter.

It's the Asciidoctor.js API that is currently lacking. I think that's what you meant.